fuzz/Cargo.toml

```diff
@@ -8,6 +8,11 @@ edition = "2021"
 [package.metadata]
 cargo-fuzz = true
 
+[features]
+# when enabling the "counters" features, the CLVM interpreter is instrumented to
+# collect counters about the programs it executes
+counters = ["clvmr/counters"]
+
 [dependencies]
 libfuzzer-sys = { workspace = true }
 clvmr = { workspace = true }
```

Comment:
I would expect you can add `features = ["counters"]` here, to always depend on `clvmr` with `counters` emabled

Reply:
Done